### ISSUE TITLE
British Petroleum (BP)

### ISSUE THREAD

**Issue by diegobot1977** [simple-icons/simple-icons#5890]
**Brand Name:** BP (British Petroleum)
**Website:** https://www.bp.com/
**Alexa rank:** 20,288
**Official resources for icon and color:** Retrieve _these_ colors - check it out:

- the first one is `#009900`, green,
- the second one is `#99cc00`, lime,
- the third one is `#ffff00`, yellow, or
- the last one is `#ffffff`, white? (see below in the BP homepage)

**Comment by fbernhart** [comment]
@diegobot1977 Can you please stop closing and opening new issues for the same icons?

Why have you closed #5849?

**Comment by diegobot1977** [comment]
because it is still waiting to request a BP icon

**Comment by fbernhart** [comment]
@diegobot1977 Once you've opened an issue, *please* just leave it open.

Don't closed it and open a new issue. Because this will confuse everybody and create more work.

**Comment by ericcornelissen** [comment]
Closing this in favor of #5849. @diegobot1977, please open issues for icon requests only once in the future.

> because it is still waiting to request a BP icon

Please note that this is an open source project run by volunteers. We are not here to create SVGs of the brands you'd like us to create.
